lib/nifty-report/report.rb

```diff
@@ -26,6 +26,9 @@ def csv
     end
 
     def email_to(email_address)
+
+      wat = "aaaaaaaaaahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh"
```

Comment:
Useless assignment to variable - wat.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Line is too long. [118/80]